client/index.html

```diff
@@ -2,9 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
But you didn't include `vite.svg` in the public folder?

Reply:
Eeep! I forgot to add `vite.svg` back in!